lib/commands/test-code-coverage.js

```diff
@@ -0,0 +1,160 @@
+'use strict';
+
+let fs = require('fs');
+let path = require('path');
+let chalk = require('chalk');
+let RSVP = require('rsvp');
+
+function camelize(str) {
+  str = str.split('-').map((str) => str.charAt(0).toUpperCase() + str.slice(1)).join('');
+  return str.charAt(0).toLowerCase() + str.slice(1);
+}
+
+module.exports = {
+
+  name: 'test-code-coverage',
+  description: 'Test the generated code coverage against given goals.',
+  works: 'insideProject',
+
+  availableOptions: [
+    {
+      name: 'coverage-folder',
+      type: String,
+      aliases: ['i'],
+      default: './coverage',
+      description: 'Directory of the coverage report'
+    },
+    {
+      name: 'target-lines',
+      type: Number,
+      aliases: ['tl'],
+      description: 'The target percentage of lines to reach, e.g. 50 = 50% coverage minimum'
+    },
+    {
+      name: 'target-statements',
+      type: Number,
+      aliases: ['ts'],
+      description: 'The target percentage of statements to reach, e.g. 50 = 50% coverage minimum'
+    },
+    {
+      name: 'target-functions',
+      type: Number,
+      aliases: ['tf'],
+      description: 'The target percentage of functions to reach, e.g. 50 = 50% coverage minimum'
+    },
+    {
+      name: 'target-branches',
+      type: Number,
+      aliases: ['tb'],
+      description: 'The target percentage of branches to reach, e.g. 50 = 50% coverage minimum'
+    }
+  ],
+
+  /**
+   * Tries to extend `availableOptions` with globals from config.
+   *
+   * @public
+   * @method beforeRun
+   * @return {Void}
+   */
+  beforeRun() {
+    this._super.apply(this, arguments);
+
+    // try to read global options from `config/coverage.js`
```

Comment:
Use the util at lib/config.js to get the configuration and and the project's configPath for the location of the config (see https://github.com/kategengler/ember-cli-code-coverage/blob/master/index.js#L142)